app/src/main/java/com/example/giphyapplication/composable/widgets/ErrorStateView.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.giphyapplication.composable.widgets
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+fun ErrorStateView(
+    modifier: Modifier = Modifier,
+    onAction: () -> Unit
+) {
+
+}
```

Comment:
This is your error state and it should contain a retry button or text so the user can see some visuals when an error occurs.